### DISCUSSION TITLE
How to override portal services?

### DISCUSSION THREAD

**Discussion by nishitsureka** [pimcore/pimcore#15947] [General]
Hi,
I tried to override portal engine services but unable to do so.
I have created Dataobject data pool page to list all the product data.
But somehow it is showing only published product in data pool page.
So i tried to override the services to disable published code.
but it is not overriding.

so, simply is it possible to override services of portal engine?

**Comment by kingjia90** [comment]
Closing as the same question being posted twice see https://github.com/pimcore/pimcore/discussions/15961
